packages/@sanity/migrate/src/runner/run.ts

```diff
@@ -56,13 +56,13 @@ export async function* toFetchOptionsIterable(
   }
 }
 
-export async function run(config: MigrationRunnerConfig, migration: Migration) {
+export async function* run(config: MigrationRunnerConfig, migration: Migration) {
   const stats: MigrationProgress = {
     documents: 0,
     mutations: 0,
     pending: 0,
     queuedBatches: 0,
-    completedTransactions: [],
+    completedTransactionsLength: 0,
```

Comment:
this wasn't strictly needed for this PR, but we didn't use the array for anything else than reading it's length, so thought I'd might as well free up some memory.